recipe/meta.yaml

```diff
@@ -45,16 +45,17 @@ requirements:
 
 test:
   requires:
-    - curio  # [unix]
-    - nbformat
-    - nose >=0.10.1
-    - numpy
-    - pygments
-    - requests
-    - testpath
-    - ipykernel
-    - trio
-    - matplotlib !=3.2.0
+    # TODO: restore after pypy migration
+    # - curio  # [unix]
+    # - ipykernel
+    # - matplotlib
+    # - nbformat
+    # - nose >=0.10.1
+    # - numpy
+    # - pygments
+    # - requests
+    # - testpath
+    # - trio
```

Comment:
Is there a way for us to keep these for `cpython` @isuruf?

Reply:
See https://conda-forge.org/docs/maintainer/knowledge_base.html#pypy-builds